### DISCUSSION TITLE
Possible to switch rigid body to deformable body between steps?

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2172] [Q&A]
Hello Radioss expert

I am wandering is there any possibility to switch rigid body to deformable body between steps in Radioss? For example, I have two steps. In the 1st step the parts are defined as rigid but in the 2nd step convert them to deformable? Thanks for your help.

Regards
Lee

**Comment by sebastienVilleneuve** [comment]
Hi @li2qn 

I think you should use /RBODY/OFF option in 2.rad file ( be careful you need to list rigid main node(s) in this card)
https://help.altair.com/hwsolvers/rad/topics/solvers/rad/rbody_engine_r.htm?zoom_highlight=%2FRBODY%2FOFF

I hope this will help

Regards,
Sébastien
